ansible-scylla-node/tasks/upgrade/main.yml

```diff
@@ -101,12 +108,13 @@
   ansible.builtin.fail:
     msg: "Scylla was not detected."
   when: scylla_detected is not defined
-
+  any_errors_fatal: True
 
 # Scylla installation validation
 - name: Check existing Scylla installation operating system incompatibility
   ansible.builtin.fail:
     msg: "{{ ansible_facts['distribution'] }} {{ ansible_facts['distribution_version'] }} is not supported."  
+  any_errors_fatal: True
```

Comment:
Why not to put all these tasks in a `block` and define `any_errors_fatal: True` once instead repeating it over and over?

Reply:
It will make sense to put this in a separate patch because it is going to be a lot of diff and a little functional change in it. Therefore better not to mix it with other (functional) patches.